apis/postgresql/v1alpha1/postgresql_standalone_operator_config_types.go

```diff
@@ -66,6 +70,13 @@ type PostgresqlStandaloneOperatorConfig struct {
 	Status PostgresqlStandaloneConfigStatus       `json:"status,omitempty"`
 }
 
+// PersistenceSpec contains default PVC settings.
+type PersistenceSpec struct {
+	// storageClassName is the name of the StorageClass required by the claim.
+	StorageClassName *string                             `json:"storageClassName,omitempty"`
+	AccessModes      []corev1.PersistentVolumeAccessMode `json:"accessModes,omitempty"`
```

Comment:
The PVC creation fails if the AccessModes is not given (it needs at least one). (Unlike storage class name, which is optional)
Questions/Options: 
- Should we mark this access mode as required? That way we can rely that the cluster-admin has given one
- If not required, should we default to `RWO` in the code?
- If not required, should we fail the provisioning? (we have not implemented failure conditions yet!)

@Kidswiss